lib/util/Ordering.ts

```diff
@@ -6,48 +6,79 @@ import { TermTransformer } from '../transformers/TermTransformer';
 import { TypeAlias, TypeURL } from './Consts';
 import type { ITimeZoneRepresentation } from './DateTimeHelpers';
 import { toUTCDate } from './DateTimeHelpers';
+import * as Err from './Errors';
 import type { ISuperTypeProvider, SuperTypeCallback, TypeCache, GeneralSuperTypeDict } from './TypeHandling';
 import { getSuperTypeDict } from './TypeHandling';
 
 // Determine the relative numerical order of the two given terms.
 /**
  * @param enableExtendedXSDTypes System will behave like when this was true. @deprecated
  */
-export function orderTypes(termA: RDF.Term | undefined, termB: RDF.Term | undefined, isAscending: boolean,
+export function orderTypes(termA: RDF.Term | undefined, termB: RDF.Term | undefined,
```

Comment:
`orderTypes` is an exported function, can we just change the behavior like this? (requires major release?)

Reply:
Let's just do a major update then :-)

The `isAscending` option felt a bit weird to me, and complicated the implementation too much, so I removed it.